packages/IMAPClient-Tests.package/ICAddressBookTest.class/instance/testFindNonExistentEntryRaisesError.st

```diff
@@ -1,4 +1,4 @@
 find entries
 testFindNonExistentEntryRaisesError
 
-	self should: [self addressBook find: self emailAddress] raise: Error
\ No newline at end of file
+	self should: [self addressBook find: self emailContact] raise: Error
```

Comment:
whitespace before and after square brackets

Reply:
resolved by the style guide